gpcontrib/gg_tables_tracking/sql/gg_tables_tracking.sql

```diff
@@ -0,0 +1,149 @@
+
+--Empty track is expected
+SELECT relname, size, state, segid, relkind, relam
+FROM gg_tables_tracking.tables_track;
+
+SELECT gg_tables_tracking.tracking_register_schema('gg_tables_tracking');
+
+-- 5. Test data extending event. Bloom should capture it.
+INSERT INTO gg_tables_tracking.tracking_t1 SELECT generate_series(1,100000);
+SELECT relname, size, state, segid, relkind, relam
+FROM gg_tables_tracking.tables_track;
+
+-- 6. Dropping table. The track shows only relfilenodes without names and other additional info with status 'd'.
+DROP TABLE gg_tables_tracking.tracking_t1;
+SELECT relname, size, state, segid, relkind, relam
+FROM gg_tables_tracking.tables_track;
+
+-- 8. Test actions on commit and rollback
+CREATE TABLE gg_tables_tracking.tracking_t1 (i INT)
+WITH (appendonly=true, orientation=column) DISTRIBUTED BY (i);
+INSERT INTO gg_tables_tracking.tracking_t1 SELECT generate_series(1,100000);
+
+-- If the wrapping transaction rollbacks, the Bloom filter is not cleared up.
+BEGIN;
+SELECT relname, size, state, segid, relkind, relam
+FROM gg_tables_tracking.tables_track;
+ROLLBACK;
+
+-- If commits, filter is cleared.
+BEGIN;
+SELECT relname, size, state, segid, relkind, relam
+FROM gg_tables_tracking.tables_track;
+COMMIT;
+SELECT relname, size, state, segid, relkind, relam
+FROM gg_tables_tracking.tables_track;
+
+-- 9. Test repetitive track call within the same transaction. All the
+-- calls should return the same relation set.
+INSERT INTO gg_tables_tracking.tracking_t1 SELECT generate_series(1,10000);
+BEGIN;
+SELECT relname, size, state, segid, relkind, relam
+FROM gg_tables_tracking.tables_track;
+
+CREATE TABLE gg_tables_tracking.tracking_t2 (j BIGINT) DISTRIBUTED BY (j);
+INSERT INTO gg_tables_tracking.tracking_t2 SELECT generate_series(1,10000);
+INSERT INTO gg_tables_tracking.tracking_t1 SELECT generate_series(1,10000);
+
+SELECT relname, size, state, segid, relkind, relam
+FROM gg_tables_tracking.tables_track;
+ROLLBACK;
+SELECT relname, size, state, segid, relkind, relam
+FROM gg_tables_tracking.tables_track;
+
+-- 10. Test relkind filtering.
+INSERT INTO gg_tables_tracking.tracking_t1 SELECT generate_series(1,100000);
+CREATE INDEX ON gg_tables_tracking.tracking_t1(i);
+
+-- Want to see index and block dir relation.
+SELECT gg_tables_tracking.tracking_register_schema('pg_aoseg');
+SELECT gg_tables_tracking.tracking_set_relkinds('o,i');
+
+SELECT  size, state, segid, relkind, relam
+FROM gg_tables_tracking.tables_track;
+
+DROP TABLE gg_tables_tracking.tracking_t1;
+
+-- Set empty relkinds. The track result set should be empty.
+SELECT gg_tables_tracking.tracking_set_relkinds('');
+
+SELECT gg_tables_tracking.tracking_trigger_initial_snapshot();
+
+SELECT  size, state, segid, relkind, relam
+FROM gg_tables_tracking.tables_track;
+
+-- Clean up
+SELECT gg_tables_tracking.tracking_unregister_db();
+
+\c contrib_regression;
+DROP DATABASE tracking_db1;
+-- start_ignore
+\! gpconfig -r shared_preload_libraries
```

Comment:
Don't touch other extensions in `shared_preload_libraries`!
For example
```suggestion
\! gpconfig -c shared_preload_libraries -v "$(psql -At -c "SELECT replace(current_setting('shared_preload_libraries'), 'gg_tables_tracking', '')" postgres)" > /dev/null
```